jenkins-builds/7.0.UP00-19.06-gpu

```diff
@@ -0,0 +1,81 @@
+ CMake-3.14.5.eb
+ CP2K-6.1-CrayGNU-19.06-cuda-10.1.eb                --set-default-module
+ CPMD-4.1-CrayIntel-19.06.eb                        --set-default-module
+ dask-1.2.2-CrayGNU-19.06-python3.eb                --set-default-module
+ GREASY-19.03-cscs-CrayGNU-19.06.eb                 --set-default-module
+ GROMACS-2018.6-CrayGNU-19.06-cuda-10.1.eb          --set-default-module
+ GROMACS-2019.1-CrayGNU-19.06-cuda-10.1.eb
+ GSL-2.5-CrayGNU-19.06.eb                           --set-default-module
+ GSL-2.5-CrayCCE-19.06.eb
+ GSL-2.5-CrayIntel-19.06.eb
+ HDFView-2.14.eb                                    --set-default-module
+ h5py-2.8.0-CrayGNU-19.06-python2-parallel.eb
+ h5py-2.8.0-CrayGNU-19.06-python2-serial.eb
+ h5py-2.8.0-CrayGNU-19.06-python3-parallel.eb       --set-default-module
+ h5py-2.8.0-CrayGNU-19.06-python3-serial.eb
+ Horovod-0.16.0-CrayGNU-19.06-tf-1.12.0.eb          --set-default-module
+ HPX-1.2.0-CrayGNU-19.06.eb                         --set-default-module
+ IDL-8.5.1.eb                                       --set-default-module
+ ipykernel-4.8.2-CrayGNU-19.06-python2.eb
+ ipyparallel-6.2.3-CrayGNU-19.06-python3.eb
+ jupyterlab-0.35.2-CrayGNU-19.06.eb
+ jupyterhub-0.9.6-CrayGNU-19.06.eb
+ jupyterhub-1.0.0-CrayGNU-19.06.eb                  --set-default-module
+ LAMMPS-22Aug2018-CrayGNU-19.06-cuda-10.1.eb        --set-default-module
+ Lmod-7.8.2.eb                                      --set-default-module --hidden
+ magma-2.4.0-CrayIntel-19.06-cuda-10.1.eb           --set-default-module
+ NAMD-2.13-CrayIntel-19.06-cuda-10.1.eb             --set-default-module
+ NCL-6.4.0.eb                                       --set-default-module
+ NCO-4.7.5-CrayGNU-19.06.eb                         --set-default-module
+ ncview-2.1.7-CrayGNU-19.06.eb                      --set-default-module
+ netcdf-python-1.4.1-CrayGNU-19.06-python2.eb
+ netcdf-python-1.4.1-CrayGNU-19.06-python3.eb       --set-default-module
+ ParaView-5.6.0-EGL-CrayGNU-19.06.eb                --set-default-module
+ PLUMED-2.5.1-CrayGNU-19.06.eb                      --set-default-module
+ PLUMED-2.5.1-CrayIntel-19.06.eb                    --set-default-module
+ pycuda-2018.1.1-CrayGNU-19.06-python2-cuda-10.1.eb
+ pycuda-2018.1.1-CrayGNU-19.06-python3-cuda-10.1.eb --set-default-module
+ PyExtensions-2.7.15.6-CrayGNU-19.06.eb
+ PyExtensions-3.6.5.6-CrayGNU-19.06.eb
+ Python-bare-3.6.2.eb                               --hidden
+ QuantumESPRESSO-6.3-CrayIntel-19.06.eb             --set-default-module
+ Theano-1.0.2-CrayGNU-19.06-cuda-10.1-python3.eb
+ Theano-1.0.2-CrayGNU-19.06-cuda-10.1-python3.eb    --set-default-module
+ Spark-2.3.1-CrayGNU-19.06-Hadoop-2.7.eb            --set-default-module
+ VASP-5.4.4-CrayIntel-19.06-cuda-10.1.eb            --set-default-module
+ VMD-1.9.3-egl.eb
+ VMD-1.9.3-ogl.eb
+ VTK-8.2.0-EGL-CrayGNU-19.06-python3.eb             --set-default-module
+ CubeGUI-4.4.3.eb                                   --set-default-module
```

Comment:
tools should go to a particular install path: `${apps}/UES/jenkins/7.0.UP00/ARCH/PE19.06/easybuild`